Update personalization-csp.md

docupdates opened the pull request:
Adds note about WinPro compatibility with desktop image url

<!-- 
Fill out the following information to help us review this pull request. 
You can delete these comments once you are done.
-->
<!-- 
## Description

If your changes are extensive:
- Uncomment this heading and provide a brief description here.
- List more detailed changes below under the changes heading.
-->

## Why
DesktopImageURL reports as not compatible in Intune for Windows 10 Pro, after discussion with support it seems there's a relevant internal note about it saying it's not compatible with Windows 10 Pro. This is supported by the UI text in Intune excluding Pro:
![image](https://github.com/MicrosoftDocs/windows-itpro-docs/assets/134548865/60dc628c-1cec-412c-86c6-eadf92171f07)

The Intune docs point to this being the CSP that backs the setting above: https://learn.microsoft.com/en-us/mem/intune/configuration/device-restrictions-windows-10#personalization

I would simply have suggested it be listed as incompatible, but have found contradictory information from an MS employee here ([Lu Dai-MSFT](https://learn.microsoft.com/en-us/users/na/?userid=c9083ec3-b3f6-48ad-a978-5924127a5d72)):

[Windows 10 MDM - Personalization-csp, DesktopImageUrl setting failing.](https://learn.microsoft.com/en-us/answers/questions/622758/windows-10-mdm-personalization-csp-desktopimageurl)

They state the following

_It works in Windows 10 Pro and Windows 10 Pro in S mode if SetEduPolicies in SharedPC CSP is set._ 

I'm not in a position to test that however - but a simple YES on compatibility here isn't quite accurate. I might be able to convince support to give me the internal article ID to pass it on to you if you need it for validation on the compatibility.

<!--
- Briefly describe _why_ you made this pull request.
- If this pull request relates to an issue, provide the issue number or link.
- If this pull request closes an issue, use a keyword (`Closes #123`).
  - Using a keyword will ensure the issue is automatically closed once this pull request is merged.
  - For more information, see [Linking a pull request to an issue using a keyword](https://docs.github.com/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
-->


## Changes

- Adds compatibility note for DesktopImageURL regarding Windows Pro compatibility.
- 
<!--
- Briefly describe or list _what_ this PR changes.
- Share any important highlights regarding your changes, such as screenshots, code snippets, or formatting.
-->

<!--
Thanks for contributing to Microsoft technical content!

Here are some resources that might be helpful while contributing:
- [Microsoft Docs contributor guide](https://learn.microsoft.com/contribute/)
- [Docs Markdown reference](https://learn.microsoft.com/contribute/markdown-reference)
- [Microsoft Writing Style Guide](https://learn.microsoft.com/style-guide/welcome/)
-->

JohanFreelancer9 commented on windows/client-management/mdm/personalization-csp.md:
```diff
@@ -88,6 +88,12 @@ This represents the status of the DesktopImage. 1 - Successfully downloaded or c
 A http or https Url to a jpg, jpeg or png image that needs to be downloaded and used as the Desktop Image or a file Url to a local image on the file system that needs to be used as the Desktop Image.
+
+> [!NOTE]
+> This setting is only compatible with Windows 10 Pro if SetEduPolicy is enabled in 
+[SharedPC](sharedpc-csp.md)
+>
+> This only applies to Windows 10 Pro and not the other editions listed
```
```suggestion
> This setting is only compatible with Windows 10 Pro if SetEduPolicy is enabled in 
[SharedPC](sharedpc-csp.md).
>
> This only applies to Windows 10 Pro and not the other editions listed.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @docupdates, kindly commit the suggested minor change. Many thanks.

docupdates commented:
> Copy review - @docupdates, kindly commit the suggested minor change. Many thanks.

Thanks for the review! Changes committed!

docupdates commented:
> @docupdates This article used to have this note for the entire CSP, but I recently automated the CSP articles, and the note was lost in translation. I've added it back, and removed your edit.

Even better, thank you!
